Nodejs/Product/Nodejs/Snippets/1033/Nodejs/parse.snippet

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>parse</Title>
+      <Description>Code Snippet for parsing a JSON string</Description>
+      <Shortcut>parse</Shortcut>
+      <SnippetTypes>
+        <SnippetType>Expansion</SnippetType>
+      </SnippetTypes>
+    </Header>
+    <Snippet>
+      <Declarations>
+        <Literal>
+          <ID>string</ID>
+          <Default>string</Default>
+        </Literal>
+      </Declarations>
+      <Code Language="Nodejs">
+        <![CDATA[JSON.parse('$string$');]]>

```

Comment:
it's rare that someone would want to parse an explicit string rather than a variable here so don't bother with the quotes.